compliance_suite/report_server.py

```diff
@@ -64,7 +67,7 @@ def start_local_server(
         http_handler = http.server.SimpleHTTPRequestHandler
         self.local_server = socketserver.TCPServer(("", port), http_handler)
         logger.info(f"Starting a local server at  http://localhost:{port}")
-        webbrowser.open(f"http://localhost:{port}")
+        webbrowser.open(f"http://localhost:{port}/report-tes-{self.versions[0]}.html")
```

Comment:
Is this how the default version to show is set? If so, I'm not sure the behavior is as you say, i.e., that the version the user specifies _first_ is the one that is shown. Rather, it would, I think, be always the smallest version, because of `versions.sort()` in the CLI module.

However, personally I'm actually not quite sure there's a lot of benefit to allow the user specifying the order of the reports (and if so, that behavior should be documented somewhere). Rather, I would always just start with the latest version.